app/lib/features/home/widgets/my_tasks.dart

```diff
@@ -68,21 +68,21 @@ class MyTasksSection extends ConsumerWidget {
                           (task) => TaskEntry(
                             task: task,
                             showBreadCrumb: true,
-                            onDone: () {
-                              EasyLoading.showToast(
-                                L10n.of(context).markedAsDone,
-                                toastPosition: EasyLoadingToastPosition.bottom,
-                              );
-                            },
+                            onDone: () => onTaskDone(context),
```

Comment:
this exporting seems unnecessary ....
```suggestion
                            onDone: () => EasyLoading.showToast(L10n.of(context).markedAsDone);,
```

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1580/commits/37ecf0ea45e677c1578c8e65c3d87e73b94dc31b